rails-time-track/db/seeds.rb

```diff
@@ -5,3 +5,72 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require 'faker'
+
+3.times do
+  User.create(
+    name: Faker::Name.name,
+    email: Faker::Internet.email,
+    # role: Faker::Job.position,
+    role: "Owner",
+    rate: Faker::Number.between(1000,2000)
+  )
+end
+
+3.times do
+  User.create(
+    name: Faker::Name.name,
+    email: Faker::Internet.email,
+    role: "Manager",
+    rate: Faker::Number.between(1000,2000)
+  )
+end
```

Comment:
In this case, maybe 2 managers could be enough. Also a fixed rate could be useful. So I suggest to create manually (avoid using "times"), to managers with specific rates.

Reply:
Thanks. We just have 2 manager now.